src/tests/jsonLogicFixtures.js

```diff
@@ -0,0 +1,1098 @@
+  'x-jsf-logic': {
+    validations: {
+      a_bigger_than_b: {
+        errorMessage: 'A must be bigger than B',
+        rule: {
+          '>': [{ var: 'field_a' }, { var: 'field_b' }],
+        },
+      },
+      is_even_number: {
+        errorMessage: 'B must be even',
+        rule: {
+          '===': [{ '%': [{ var: 'field_b' }, 2] }, 0],
+        },
+      },
+    },
+  },
+};
+
+export const schemaWithComputedAttributes = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+    field_b: {
+      type: 'number',
+      'x-jsf-computedAttributes': {
+        title: 'This is {{a_times_two}}!',
+        value: 'a_times_two',
+        description: 'This field is 2 times bigger than field_a with value of {{a_times_two}}.',
+      },
+    },
+  },
+  required: ['field_a', 'field_b'],
+  'x-jsf-logic': {
+    computedValues: {
+      a_times_two: {
+        rule: {
+          '*': [{ var: 'field_a' }, 2],
+        },
+      },
+    },
+  },
+};
+
+export const nestedFieldsetWithValidationSchema = {
+  properties: {
+    field_a: {
+      type: 'object',
+      'x-jsf-presentation': {
+        inputType: 'fieldset',
+      },
+      properties: {
+        child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['child_greater_than_10'],
+        },
+      },
+      required: ['child'],
+      'x-jsf-logic': {
+        validations: {
+          child_greater_than_10: {
+            errorMessage: 'Must be greater than 10!',
+            rule: {
+              '>': [{ var: 'child' }, 10],
+            },
+          },
+        },
+      },
+    },
+  },
+  required: ['field_a'],
+};
+
+export const validatingTwoNestedFieldsSchema = {
+  properties: {
+    field_a: {
+      type: 'object',
+      'x-jsf-presentation': {
+        inputType: 'fieldset',
+      },
+      properties: {
+        child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['child_greater_than_10'],
+        },
+        other_child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['greater_than_child'],
+        },
+      },
+      required: ['child', 'other_child'],
+      'x-jsf-logic': {
+        validations: {
+          child_greater_than_10: {
+            errorMessage: 'Must be greater than 10!',
+            rule: {
+              '>': [{ var: 'child' }, 10],
+            },
+          },
+          greater_than_child: {
+            errorMessage: 'Must be greater than child',
+            rule: {
+              '>': [{ var: 'other_child' }, { var: 'child' }],
+            },
+          },
+        },
+      },
+    },
+  },
+  required: ['field_a'],
+};
+
+export const twoLevelsOfJSONLogicSchema = {
+  properties: {
+    field_a: {
+      type: 'object',
+      'x-jsf-presentation': {
+        inputType: 'fieldset',
+      },
+      properties: {
+        child: {
+          type: 'number',
+          'x-jsf-requiredValidations': ['child_greater_than_10'],
```

Comment:
question: I tried the following variant where I use `peek_to_nested` here:

<details>

```json
export const twoLevelsOfJSONLogicSchema = {
  properties: {
    field_a: {
      type: 'object',
      'x-jsf-presentation': {
        inputType: 'fieldset',
      },
      properties: {
        child: {
          type: 'number',
          'x-jsf-requiredValidations': ['child_greater_than_10', 'peek_to_nested'],
        },
      },
      required: ['child'],
      'x-jsf-logic': {
        validations: {
          child_greater_than_10: {
            errorMessage: 'Must be smaller than 50!',
            rule: {
              '<': [{ var: 'child' }, 50],
            },
          },
        },
      },
    },
    field_b: {
      type: 'number',
      'x-jsf-requiredValidations': ['validation_parent'],
    },
  },
  'x-jsf-logic': {
    validations: {
      validation_parent: {
        errorMessage: 'Must be greater than 10!',
        rule: {
          '>': [{ var: 'field_b' }, 10],
        },
      },
      peek_to_nested: {
        errorMessage: 'child must be greater than 15!',
        rule: {
          '>': [{ var: 'field_a.child' }, 15],
        },
      },
    },
  },
  required: ['field_a', 'field_b'],
};
```

</details>

And then run the respective test `"Validate a field and a nested field together"`. As expected the test failed:

```
    JSON Schema invalid! Error: Validation "peek_to_nested" required for "child" doesn't exist.
```

I think we could have an extra test to ensure this will always fail for sanity check. We already have for root fields, but not for fieldsets.